Homeworks/Homework2/src/tests/demoblaze_test.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from selenium import webdriver
+
+from Homeworks.Homework2.src.main.cart_page import CartPage
+from Homeworks.Homework2.src.main.config import user_name, user_password
+from Homeworks.Homework2.src.main.demoblaze_page import DemoBlazePage
+from Homeworks.Homework2.src.main.item_page import ItemPage
+
+
+# To initialize WebDriver
+@pytest.fixture
```

Comment:
fixtures are usually kept separately in conftest.py file

Reply:
thanks, moved to conftest.py